### ISSUE TITLE
Better Combat Animation Problem

### ISSUE THREAD

**Issue by WARDEN-Xanni** [ToCraft/woodwalkers-mod#97]
### Mod loader

Forge

### Minecraft version

1.20.1

### Woodwalkers version

4.5

### CraftedCore version

4.2.3

### Modloader version

0.15.11

### The latest.log file

https://mclo.gs/8434gHE

### Issue description

Better Combat first person animations no longer work.

### Steps to reproduce

1. Grab a weapon
2. Swing weapon
3. Witness lack of animation
4. :(

**Comment by ToCraft** [comment]
Yeah, that's actually intended, since before 4.5, animations caused you to see your own head. I might be able to make the animations work when the player isn't morphed, but animations while being morphed probably won't work for now.

**Comment by ToCraft** [comment]
I have now implemented with 9b8fd9edf6a0c129d87d0768e81f895146032992 that the animation is only suppressed when you are morphed.
